docs/read_next.rst

```diff
@@ -1,3 +1,29 @@
-What to read next
+What to Read Next
 =================
 
+Depending on your role, certain aspects of CumulusCI may pertain to you more than others.
+Use this section as a guide for which parts of the documentation are most likely to be relevant to you.
+
+For everyone
+------------
+
+Developers
+----------
+    * Tutorial
+
+Quality Engineers
+-----------------
+Text here.
```

Comment:
How can we track these to-do items so they don't get lost?

Reply:
Status and Notes columns in the quip doc with the outline?

Reply:
I think we could: adhere to a specific string we use to denote a to-do in the docs (e.g. #TODO), make work items for ourselves in gus under a docs epic, or a new quip doc with kanban board.

Work items seem bulky, but I lean towards the use something like `#TODO` because it is easily searchable by all.

Reply:
I like using TODO. I did that in a couple of places in mine where references will need to be added once the referenced section exists.

Reply:
I'm fine with that.

Reply:
@davidmreed @davisagli - This extension mades them quite easy to track as well: https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree